frontend/src/components/FollowUp.vue

```diff
@@ -0,0 +1,96 @@
+<template>
```

Comment:
I don't think this page is nessecary. we can do same login in the create flist page.

Reply:
this page was requested to poll on the job id to know if the job was failed or not